code/modules/modular_computers/computers/item/tablet.dm

```diff
@@ -190,7 +190,7 @@
 /// A simple proc to set the ringtone from a pda.
 /obj/item/modular_computer/tablet/proc/update_ringtone(new_ringtone)
 	if(!istext(new_ringtone))
-		return
+		CRASH("Expected a text for ringtone, received [new_ringtone]")
```

Comment:
Ugh, this SHOULD be an error, but looks like everything does it right now. Either fix that or change this back to a return.